### PR TITLE
Update documentation, fix benchmark

### PR DESCRIPTION
**Pull request by Christopher-Chianelli**
### Referenced pull requests

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* for a <b>pull request build</b> please add comment: <b>Jenkins retest this</b>
* for a <b>full downstream build</b> please add comment: <b>Jenkins run fdb</b>
* for a <b>compile downstream build</b> please add comment: <b>Jenkins run cdb</b>
* for a <b>full production downstream build</b> please add comment: <b>Jenkins execute product fdb</b>
* for an <b>upstream build</b> please add comment: <b>Jenkins run upstream</b>
</details>

### CODE REVIEW

File: optaweb-employee-rostering-benchmark/src/main/java/org/optaweb/employeerostering/OptaWebEmployeeRosteringBenchmarkApplication.java [hunk 1]
```diff
@@ -53,7 +53,8 @@
     private final Logger logger = LoggerFactory.getLogger(getClass());
 
     public static void main(String[] args) {
-        Quarkus.run(args);
+        Quarkus.run(OptaWebEmployeeRosteringBenchmarkApplication.class,
+                args);
```
Comment (triceo) [inline optaweb-employee-rostering-benchmark/src/main/java/org/optaweb/employeerostering/OptaWebEmployeeRosteringBenchmarkApplication.java:57]: Your IDE keeps doing this. :-) These should be on one line.

**Review by triceo — APPROVED** [review]
Soft suggestion inside.

File: optaweb-employee-rostering-benchmark/src/main/java/org/optaweb/employeerostering/OptaWebEmployeeRosteringBenchmarkApplication.java (hunk 1 above)
Comment (Christopher-Chianelli) [reply optaweb-employee-rostering-benchmark/src/main/java/org/optaweb/employeerostering/OptaWebEmployeeRosteringBenchmarkApplication.java:57]: Na, it me trying to keep it to 80 so it fits my screen

Comment (triceo) [reply optaweb-employee-rostering-benchmark/src/main/java/org/optaweb/employeerostering/OptaWebEmployeeRosteringBenchmarkApplication.java:57]: Are you using a terminal from 1981?
We agreed on line length of 120.

Comment (Christopher-Chianelli) [reply optaweb-employee-rostering-benchmark/src/main/java/org/optaweb/employeerostering/OptaWebEmployeeRosteringBenchmarkApplication.java:57]: No, and IDE with large font which make it effectively 80; I can change the line

Comment (triceo) [reply optaweb-employee-rostering-benchmark/src/main/java/org/optaweb/employeerostering/OptaWebEmployeeRosteringBenchmarkApplication.java:57]: Ah, I see. 
If the default length is causing issues, perhaps we should discuss that as a team? 
Feel free to bring it up,  no point in enforcing something which may be needlessly painful.

**Pull request merged**